en/config_heli/README.md

```diff
@@ -0,0 +1,87 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
```

Comment:
What do we mean by well-supported here? I mean I presume a helicopter supports all the same modes as multicopter in the same way - including offboard?

What other types of frames are there?
What features might be missing?

Reply:
Modes yes.
In terms of frames, there's e.g. coaxial or dual rotor types. Features include RPM governor or autorotation.